source/helpers/get-rgh-issue-url.ts

```diff
@@ -0,0 +1,3 @@
+export default function getRghIssueUrl(issueId: number | string): string {
+	return 'https://github.com/sindresorhus/refined-github/issues/' + issueId.toString();
+}
```

Comment:
Can you merge this into the helper file added by https://github.com/sindresorhus/refined-github/pull/4387? 

This function can be used in background.ts, and the **link** function can be used in options.tsx